### DISCUSSION TITLE
Limits of Begin Data

### DISCUSSION THREAD

**Discussion by Shaun-Regenbaum** [beginner-corp/begin-community#86] [General] [answered]
I've been looking at Begin Data and am a bit confused.
What is the pricing structure, what are the limits, etc?
Also, can I use it for user authentican, data, and everything else?

**Comment by brianleroux** [comment, marked as answer]
Hi @Shaun-Regenbaum … pricing is FREE currently. When we make paid accounts generally available there will be more info on pricing and quotas. For the moment the Begin free tier apps are shared on an AWS account which means you are sharing the standard resources of 1 AWS account between a maximum of 1000 other apps. Most of the time this won't be an issue but this does mean your free tier app can suffer throttles if you have a noisy neighbor on the same account. So the limits are the published limits and quotas for the AWS primitives supported by Begin. These are API Gateway, Lambda, DynamoDB, S3, SNS and EventBridge respectively. 

> Also, can I use it for user authentican, data, and everything else?

Yes! We have a tutorial for implementing oAuth with Begin Data at https://learn.begin.com using Github as a provider. You can store anything in Begin Data that you would with DynamoDB.
